db/dcrpg/pgblockchain.go

```diff
@@ -448,7 +465,16 @@ func NewChainDB(dbi *DBInfo, params *chaincfg.Params, stakeDB *stakedb.StakeData
 		tpUpdatePermissions[g] = new(trylock.Mutex)
 	}
 
+	// If a query timeout is not set (i.e. zero), default to 24 hrs for
```

Comment:
Since `defaultPGQueryTimeout float64 = 86400.0` exists it therefore means that `dbi.QueryTimeout` will be zero or less than zero if a user did set it to that value.

Infact this feels more appropriate:
```golang
if queryTimeout < 1 {
	queryTimeout = 24 * time.Hour
}
```

Reply:
I didn't consider what would happen if the user set it to a negative number.  But I think I will validate the input in config.go instead of here.  That way there aren't the two constants.